### PR TITLE
Finial Manifest update

### PR DESCRIPTION
**Pull request by HarishH-DELL**
# Description
Finial Manifest update

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1091 |

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

### CODE REVIEW

File: bundle/manifests/dell-csm-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -3599,61 +3606,6 @@ spec:
   provider:
     name: Dell Technologies
     url: https://github.com/dell/csm-operator
-  relatedImages:
```
Comment (rajendraindukuri) [inline bundle/manifests/dell-csm-operator.clusterserviceversion.yaml:3602]: @HarishH-DELL This section should be there. Please add it manually if deleted during building manifests

File: bundle/manifests/connectivity-client-docker-k8s-cert_v1_secret.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+data:
+  cert.pem: ""
+kind: Secret
+metadata:
+  name: connectivity-client-docker-k8s-cert
+type: Opaque
```
Comment (panigs7) [inline bundle/manifests/connectivity-client-docker-k8s-cert_v1_secret.yaml:7]: Revert this file. We don't require this file while bundling.

File: bundle/manifests/dell-csm-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -1271,9 +1271,16 @@ metadata:
...
     description: Easily install and manage Dell’s CSI Drivers and CSM
-    operators.operatorframework.io/builder: operator-sdk-v1.32.0
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    operators.operatorframework.io/builder: operator-sdk-v1.34.1
```
Comment (panigs7) [inline bundle/manifests/dell-csm-operator.clusterserviceversion.yaml:1283]: Are you using operator-sdk version 1.34.1?

File: bundle/manifests/dell-csm-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -3599,61 +3606,6 @@ spec:
...
-    name: resizer
-  - image: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.11.0
-    name: externalhealthmonitorcontroller
-  - image: dellemc/csi-metadata-retriever:v1.7.2
-    name: metadataretriever
-  - image: docker.io/dellemc/connectivity-client-docker-k8s:1.2.3
-    name: dell-connectivity-client
-  - image: docker.io/dellemc/connectivity-cert-persister-k8s:0.11.0
-    name: cert-persister
   skips:
```
Comment (panigs7) [inline bundle/manifests/dell-csm-operator.clusterserviceversion.yaml:3609]: Revert this relateImages deletion.

File: bundle/metadata/annotations.yaml [hunk 5]
```diff
@@ -4,15 +4,11 @@ annotations:
...
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.34.1
   operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
   operators.operatorframework.io.metrics.project_layout: go.kubebuilder.io/v3
 
   # Annotations for testing.
   operators.operatorframework.io.test.mediatype.v1: scorecard+v1
   operators.operatorframework.io.test.config.v1: tests/scorecard/
-
-  # Annotations to specify supported OCP versions.
-  com.redhat.openshift.versions: v4.13-v4.14
```
Comment (panigs7) [inline bundle/metadata/annotations.yaml:18]: Remove the changes in the annotations.yaml file. Only change to be preserved is the sdk version used to generate the files.

File: deploy/operator.yaml [hunk 6]
```diff
@@ -1249,8 +1222,6 @@ spec:
     metadata:
       labels:
         control-plane: controller-manager
-      annotations:
-        storage.dell.com/CSMVersion: v1.10.0
     spec:
       containers:
       - args:
```
Comment (panigs7) [inline deploy/operator.yaml:1227]: Revert all the changes in this file

**Review by panigs7 — CHANGES_REQUESTED** [review]

**Review by panigs7 — APPROVED** [review]

**Review by rajendraindukuri — APPROVED** [review]

**Pull request merged**